changelog.txt

```diff
@@ -10,6 +10,7 @@
 * Fix - Use configured domain instead of current domain for Apple Pay verification.
 * Fix - Fatal error when deactivating the WooCommerce plugin when WCPay Subscriptions is enabled.
 * Fix - Do not create WooCommerce Payments Subscriptions when using payment methods other than WooCommerce Payments.
+* Fix - Use reports customer id for customers table. 
```

Comment:
The wordings do not sufficiently explain the changes and may benefit from a little rephrasing.